raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java

```diff
@@ -207,59 +243,76 @@ public void go(nConsumeEvent evt) {
 		if (evt.hasAttributes()) {
 			System.out.println("Published on: " + new Date(evt.getAttributes().getTimestamp()).toString());
 		}
+
 		// Print the properties
 		nEventProperties prop = evt.getProperties();
 		if (prop != null) {
 			System.out.println("Source: " + prop.get("source"));
 			System.out.println("Category: " + prop.get("category"));
 		}
-
+		
 		// download raster
-
+		// TODO remove condition? See L293.
+		
 		if (prop.get("source").equals("hhi")) {
 
-			// parse incoming message
-
-			//TODO tmp example message
-			evtData = new JSONObject("{\"category\": \"raster-forecast\", \"source\": \"hhi\", \"payload\": {\"url\":\"https://www.geomer.de/dltemp/nrw_2020010109.tif\", \"predictionStartTime\": 1578240000, \"region\": \"NRW\", \"interval\": 3600, \"creationTime\": 1590573111, \"stationId\": \"WESE\", \"predictionEndTime\": 1578412800, \"type\": \"PM10_GM1H24H\", \"unit\": \"microgm-3\"}, \"timestamp\": 1591686245}");
-			//JSONObject evtData = new JSONObject(new String(evt.getEventData()));
-
-			String category = evtData.getString("category");
-
-			Long time_stamp = evtData.getLong("timestamp"); //get timestamp and convert to format readable by geoserver regex
-			SimpleDateFormat format = new SimpleDateFormat("yyyyMMddhh");
-			String readableTime = format.format(time_stamp*1000);
-
-			JSONObject payload = evtData.getJSONObject("payload");
-			String request = payload.getString("url");
-
-			evtRegion = payload.getString("region");
-			evtPollutant = payload.getString("type");
-
-			String fileName = evtRegion.toLowerCase()+"_"+evtPollutant+"_"+readableTime;
-
-			// TODO Check if there actually is "realtime" data
-			if (category.contains("forecast")) {
-				fileName = "fc_"+fileName;
-			} else if (category.contains("realtime")) {
-				fileName = "rt_"+fileName;
-			} else {
-				System.out.println("Error: Could not determine if real time / forecast");
-				System.exit(1);
-			}
-
-
-			System.out.println("URL to raster to download: " + request);
-
-			// TODO whitelist request URLs
-
 			try {
-
-				this.downloadRaster(request, fileName);
-
-			} catch (IOException e) {
-				System.out.println("Could not download raster file");
-				e.printStackTrace();
+					
+				// parse incoming message
+				
+				RasterDownloader.evtData = new JSONObject(new String(evt.getEventData()));
+	
+				String category = evtData.getString("category");
+				SimpleDateFormat format = new SimpleDateFormat("yyyyMMddhh");
+	
+				JSONObject payload = evtData.getJSONObject("payload");
+				String request = payload.getString("url");			
+				Long time_stamp = payload.getLong("predictionStartTime"); //get timestamp and convert to format readable by geoserver regex

```

Comment:
Please use camel case notation for vars and more speaking var name:
 ```suggestion
				Long predictionStartTime = payload.getLong("predictionStartTime"); //get timestamp and convert to format readable by geoserver regex

```